tests/foreman/ui/test_oscapcontent.py

```diff
@@ -0,0 +1,161 @@
+                    self.assertIsNotNone(
+                        self.oscapcontent.search(content_name))
+
+    @tier1
+    @skip_if_bug_open('bugzilla', 1289571)
+    def test_negative_create(self):
+        """@Test: Create OpenScap content with negative values
+
+        @Feature: OpenScap - Negative Create.
+
+        @Steps:
+
+        1. Create an openscap content.
+        2. Provide all the appropriate parameters.
+
+        @Assert: Whether creating  content for OpenScap is not successful.
+        """
+        with Session(self.browser) as session:
+            for content_name in invalid_values_list(interface='ui'):
+                with self.subTest(content_name):
+                    set_context(session, org=ANY_CONTEXT['org'])
+                    session.nav.go_to_oscap_content()
+                    make_oscapcontent(
+                        session,
+                        name=content_name,
+                        content_path=self.content_path,
+                    )
+                    self.assertIsNotNone(session.nav.wait_until_element(
+                        common_locators['haserror']))
+
+    @tier1
+    def test_positive_default(self):
+        """@Test: Check whether OpenScap content exists by default.
+
+        @Feature: OpenScap - Test Default Content.
+
+        @Steps:
+
+        1. Navigate to oscap Content page.
+        2. Set Org as Any Org.
+
+        @Assert: Whether oscap content exists by default.
+        """
+        with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            session.nav.go_to_oscap_content()
+            self.assertIsNotNone(self.oscapcontent.search(
+                OSCAP_DEFAULT_CONTENT['rhel7_content']))
+            self.assertIsNotNone(self.oscapcontent.search(
+                OSCAP_DEFAULT_CONTENT['rhel6_content']))
+
+    @tier2
+    def test_positive_update(self):
+        """@Test: Update OpenScap content.
+
+        @Feature: OpenScap - Positive Update.
+
+        @Steps:
+
+        1. Create an openscap content.
+        2. Provide all the appropriate parameters.
+        3. Update the openscap content, here the Org.
+
+        @Assert: Whether creating  content for OpenScap is successful.
+        """
+        org = gen_string('alpha')
+        content_name = gen_string('alpha')
+        with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            session.nav.go_to_oscap_content()
+            make_oscapcontent(
+                session,
+                name=content_name,
+                content_path=self.content_path,
+            )
+            make_org(
+                session,
+                org_name=org,
+            )
+            self.assertIsNotNone(
+                self.org.search(org))
+            self.oscapcontent.update(content_name, content_org=org)
+            session.nav.go_to_select_org(org)
+            session.nav.go_to_oscap_content()
```

Comment:
Search functionality usually does navigation, so that line seems as redundant one